src/main/features/response/routes/how-much-paid.ts

```diff
@@ -0,0 +1,35 @@
+import * as express from 'express'
+
+import { Paths } from 'features/response/paths'
+
+import { Form } from 'forms/form'
+import { FormValidator } from 'forms/validation/formValidator'
+import { HowMuchPaid } from 'features/response/form/models/howMuchPaid'
+import User from 'idam/user'
+import { ResponseDraftMiddleware } from 'response/draft/responseDraftMiddleware'
+import { ErrorHandling } from 'common/errorHandling'
+
+function renderView (form: Form<HowMuchPaid>, res: express.Response): void {
+  const name: string = res.locals.user.forename + ' ' + res.locals.user.surname
```

Comment:
err why are we getting this name from IDAM?
We've been ignoring it up till now, and use the name entered by the claimant,
also in 99% of the cases the defendant won't have a surname from IDAM because we put their name in the firstname.

This information should be coming from the claim